integrations/zendesk/src/setup.ts

```diff
@@ -6,7 +6,7 @@ import { Triggers } from './triggers'
 
 export const register: IntegrationProps['register'] = async ({ client, ctx, webhookUrl, logger }) => {
   try {
-    await unregister({ ctx, client, webhookUrl, logger })
+    unregister && (await unregister({ ctx, client, webhookUrl, logger }))
```

Comment:
chose: declare the unregister function as `NonNullable<IntegrationProps['unregister']>` and you wont have to do that